andela/northstar-backend
# ft(Email Notification):User gets email notification on signup

### What does this PR do?
- Creates the functionality that sends a welcome email to a user at signup

### Description of tasks to be completed
- Design the email template on sendgrid
- Create a helper function that automatically sends mail when called
- Get API and other template info from sendgrid
- Call the email sender function after sending a request to the signup endpoint
- Test the endpoint for this action

### How should this be manually tested?
-  Clone or download this repository
-  Run `npm install` to install node packages
- Add the below .env files to your already existing ones
```
SEND_GRID_API=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
SENDER_EMAIL='your email'
//Set a sender email on your .env file
//Set your email on the test.js file to receive an email
``` 

- Run `npm test` to test endpoint
- Check your email. A mail will be sent to your email

### What are the relevant Pivotal Tacker stories?
[finishes #167727452](https://www.pivotaltracker.com/story/show/167727452)

###Who should take a look###
@okoroemeka
